sdks/python/apache_beam/runners/direct/evaluation_context.py

```diff
@@ -102,7 +102,8 @@ def __init__(self, side_inputs):
         list
     )  # type: DefaultDict[Optional[AppliedPTransform], List[pvalue.AsSideInput]]
     # this appears unused:
-    self._side_input_to_blocked_tasks = collections.defaultdict(list)  # type: ignore
+    self._side_input_to_blocked_tasks = collections.defaultdict(
+        list)  # type: ignore
```

Comment:
Would be good to drop this unrelated change

Reply:
Hmm... I think the linter did that to me.